ui-manchette/src/styles/manchette.css

```diff
@@ -9,17 +15,29 @@
     width: 100%;
     backdrop-filter: blur(8px);
     -webkit-backdrop-filter: blur(8px);
-    border-top: solid 1px;
-    border-right: solid 1px;
-    @apply border-black-25;
-    border-radius: 0px 0px 0px 10px;
 
-    .zoom-out:disabled {
-      opacity: 0.4;
-    }
 
-    .zoom-in:disabled {
-      opacity: 0.4;
+    .zoom-buttons {
+      display: flex;
+      align-items: center;
+      height: 100%;
+
+      button {
+        border-right: solid 1px #d3d1cf;
```

Comment:
Could we use `theme()` to avoid hardcoding a color here?

https://tailwindcss.com/docs/functions-and-directives#theme

Reply:
Sure